stl/inc/algorithm

```diff
@@ -1073,11 +1073,11 @@ namespace ranges {
                     return false;
                 }
 
-                return _Sized_ranges(_STD move(_UFirst1), _STD move(_ULast1), _STD move(_UFirst2), _STD move(_ULast2),
-                    _Count, _Pass_fn(_Pred), _Pass_fn(_Proj1), _Pass_fn(_Proj2));
+                return _Is_permutation_with_lengths(_STD move(_UFirst1), _STD move(_ULast1), _STD move(_UFirst2),

```

Comment:
I am super sorry but would `_Is_permutation_sized` and `_Is_permutation_unsized` be viable alternatives?

Reply:
I'm fine with that. (WTF did I make these plural when there's only one length?)